src/akamai/js/akamaiHD_flash.js

```diff
@@ -583,6 +601,37 @@ require("../../../html5-common/js/utils/constants.js");
         width: event.eventObject.width,
         height: event.eventObject.height,
       }
+        var objectHeight = event.eventObject.height;
+        var objectWidth = event.eventObject.width;
+        var wrapperHeight = $(videoItem).parent().height();
+        var wrapperWidth = $(videoItem).parent().width();
+
+        $(videoItem).css("height", objectHeight+'px');
+        var resizeStyles = {};
+
+        if (wrapperHeight > (objectHeight*1.05)) {
```

Comment:
The 1.1 was a buffer to just make sure 'flakiness' in rounding didn't cause the CSS changes to trigger. Needed to be sure the value of one of these variable was 'larger enough' than the other.